### DISCUSSION TITLE
Is there a way to influence the title of the research?

### DISCUSSION THREAD

**Discussion by scottiesan** [assafelovic/gpt-researcher#865] [General]
How you guys go about setting the titles?  I don't see a easy way to influence it from the api.

Cheer!

**Comment by ElishaKay** [comment]
sup @scottiesan,

Sounds like the `write_to_file` function

**Comment by adrianhensler** [comment]
You can end your query with "Title: Cheese is fun" - I tested this once, and out of one time it worked.  100% success rate.
